Configuration/Services.yaml

```
@@ -49,6 +49,13 @@ services:
         event: Extcode\Cart\Event\Order\PersistOrderEvent
         after: 'cart--order--create--persist-order--item'
 
+  Extcode\Cart\EventListener\Order\Create\PersistOrder\Taxes:
+    tags:
+      - name: event.listener
+        identifier: 'cart--order--create--persist-order--tax'
```

My syntax for the identifier is related to the class name. Can you change this?

```suggestion
        identifier: 'cart--order--create--persist-order--taxes'
```